src/core/unif.ml

```diff
@@ -109,30 +113,27 @@ let instantiation : ctxt -> meta -> term array -> term ->
         Some (Bindlib.bind_mvar vs (lift u))
   else None
 
-(** [instantiate ctx m ts u] check whether, in a problem [m[ts]=u], [m] can be
-    instantiated and, if so, instantiate it. *)
+(** [instantiate ctx m ts u] check whether, in a problem [m[ts]=u], [m] can
+   be instantiated and, if so, instantiate it. *)
```

Comment:
Comment indentation is incorrect, please be careful with this (or set up your editor).